lib/scss/components/local-footer.component.scss

```diff
@@ -0,0 +1,35 @@
+@charset 'UTF-8';
+@import '../jumpstart_ui.config';
+@import 'decanter-no-markup';
+
+.su-local-footer__social-links {
+
+  a {
+    font-size: 0;
+    font-family: "Font Awesome 5 Brands";
+
+    &::before {
+      font-size: 1.9rem;
+    }
+
+    &[href*="facebook"]::before {
```

Comment:
These are missing the colors.

Reply:
i assumed they would come from decanter

Reply:
They would have but the links are missing the css class that gives it to them.

See: https://github.com/SU-SWS/decanter/blob/v6/core/src/scss/components/local-footer/_local-footer.scss#L317

Reply:
The colors for each will have to be copied over to this file.

Reply:
that line doesnt apply any colors. it looks like its not the class. its the `<i>` tag

Reply:
S'all nested so take yer pick.

Reply:
well Personally i think the way that css is done its way too restrictive. Its basically saying you have to have an `<i>` tag or we dont apply anything.

Reply:
Also, the markup change to remove the `<i>` & `spans` likely breaks the accessibility for screen readers on this. I'll have to test.

Reply:
i tested screen readers and it was fully functional.

Reply:
The change in markup on the links as they are not in lists is breaking the spacing. This will need some more work.

<img width="1675" alt="Screen Shot 2019-11-13 at 3 01 59 PM" src="https://user-images.githubusercontent.com/550602/68811963-bac1ac00-0626-11ea-9974-0e49965a17a9.png">

Reply:
well theres more css needed. but i'm just wiring up the backend

Reply:
I can pick up this ticket again if you'd like.

Reply:
I just get frustrated when i try to use decanter's css. i try to avoid touching it. if you wanna take this or we could just create another ticket for later.

Reply:
The point is to avoid touching it. The magic is in matching the markup or selectors. I'll take it from here.

Reply:
well anything not needing css would need a template. and that would have to be in the theme.

Reply:
I duplicated the scss from decanter to work with this implementation.. have a looksee plz